sdk/keyvault/Microsoft.Azure.Management.KeyVault/src/Microsoft.Azure.Management.KeyVault.csproj

```diff
@@ -11,12 +11,11 @@
 		</Description>
 		<AssemblyTitle>Microsoft Azure Key Vault Management</AssemblyTitle>
 		<AssemblyName>Microsoft.Azure.Management.KeyVault</AssemblyName>
-		<Version>3.1.0-preview.2</Version>
+		<Version>3.1.0</Version>
```

Comment:
@isra-fel Should this be 4.1.0 (as mentioned in the conversation)?

Reply:
Sorry @nisha-bhatia my bad, description was wrong, `3.1.0` is the correct version.
Nothing to change in the code though